UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.Common/HealthCheck/AzureBlobStorageHealthCheck.cs

```diff
@@ -58,8 +58,8 @@ public async Task<HealthCheckResult> CheckHealthAsync(HealthCheckContext context
             }
             catch (Exception ex)
             {
-                logger.LogError(EventIds.AzureBlobStorageIsUnhealthy.ToEventId(), ex, "Azure blob storage is unhealthy with error {Message}", ex.Message);
-                return HealthCheckResult.Unhealthy("Azure blob storage is unhealthy", ex);
+                logger.LogError(EventIds.EventHubLoggingIsUnhealthy.ToEventId(), ex, "Health check for Azure Blob Storage threw an exception");
```

Comment:
Please use `AzureBlobStorageIsUnhealthy` event
```suggestion
                logger.LogError(EventIds.AzureBlobStorageIsUnhealthy.ToEventId(), ex, "Health check for Azure Blob Storage threw an exception");
```